rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -541,7 +553,7 @@ def train(self, training_data, cfg=None, **kwargs):
 
     # process helpers
     def _calculate_message_sim(self, X, all_Y):
-        # type: (np.ndarray, np.ndarray) -> Tuple[List[Int], List[Int]]
+        # type: (np.ndarray, np.ndarray) -> Tuple[List[int], List[int]]
```

Comment:
` # noinspection PyPep8Naming` before `def`,
and I think the first out `intent_ids` is `np.ndarray` and the second `message_sim.tolist()` is `List[float]`